### PR TITLE
Added more info to sessions popover

### LINKED ISSUE #3270: Schedule: On click show more info
On clicking on the schedule sessions currently only limited information is displayed. Please show more info here:
* Start time and End time
* Short abstract (up to 500 characters)
* Speaker info (up to 500 characters)

 screenshot from 2017-02-27 01-00-24

### PR DESCRIPTION
**Pull request by Achint08**
Fixes #3270 .
I implemented but I faced an issue:- Property short_biography wasn't set to object speaker.

screenshot from 2017-03-01 18 27 37

Kindly review and correct me. :)

### CODE REVIEW

**Comment by niranjan94** [comment]
> I implemented but I faced an issue:- Property short_biography wasn't set to object speaker.

@Achint08 can you please explain what issue this is ?

**Comment by Achint08** [comment]
Properties returned for object session.speaker .
screenshot from 2017-03-02 13 34 33 .
I couldn't get short_biography for the speaker.

**Comment by niranjan94** [comment]
@Achint08 speaker form fields can be customized ... So, sometimes there maybe short_biography, sometimes only long_biography, sometimes both, or sometimes neither. You'll have to display accordingly.

- If only `short_biography` or if both `short_biography` & `long_biography`  is there, show `short_biography`
- If only `long_biography`, show the first 100 characters of that
- if neither is there, then don't show that at all

**Comment by Achint08** [comment]
@niranjan94  I have added short_biography for the speakers. It wasn't returned by the Object speakers.
I will also execute it like what you said.

**Comment by niranjan94** [comment]
@Achint08 if it wasn't returned, then please check why it was not returned. Check the API definitions.

**Comment by Achint08** [comment]
I added condition if two or more speakers are there, it will present it like this.
screenshot from 2017-03-02 21 10 25

**Comment by Achint08** [comment]
@niranjan94 Done.

File: app/static/js/events/scheduler.js [hunk 1]
```diff
@@ -550,10 +550,33 @@ function addInfoBox($sessionElement, session) {
...
     });
-    var speakers = _.map(session.speakers, 'name');
     var content = "";
-    if(speakers.length > 0) {
-        content += "By " + _.join(speakers, ', ') + "<br><br>";
+    if(!_.isNull(session.short_abstract)) {
+        content +=  "<strong>About the session:</strong> " + session.short_abstract + "<br><br>";
+    }
+    if(session.speakers.length > 0) {
+        for(let index in session.speakers)
```
Comment (niranjan94) [inline app/static/js/events/scheduler.js]: Why is this still there ?

Comment (Achint08) [inline app/static/js/events/scheduler.js]: By mistake. Removed.

File: app/static/js/events/scheduler.js [hunk 2]
```diff
@@ -550,10 +550,32 @@ function addInfoBox($sessionElement, session) {
...
         title: session.title
     });
-    var speakers = _.map(session.speakers, 'name');
     var content = "";
-    if(speakers.length > 0) {
-        content += "By " + _.join(speakers, ', ') + "<br><br>";
+    if(!_.isNull(session.short_abstract)) {
+        content +=  "<strong>About the session:</strong> " + session.short_abstract + "<br><br>";
+    }
+    if(session.speakers.length > 0) {
```
Comment (niranjan94) [inline app/static/js/events/scheduler.js]: Why is this if statement there ? It has no purpose.

File: app/static/js/events/scheduler.js [hunk 3]
```diff
@@ -550,10 +550,32 @@ function addInfoBox($sessionElement, session) {
...
+            if(session.speakers.length === 1) {
+                content += "<strong>Speaker: </strong> " + speaker.name + "<br><br>";
+            }
+            else {
+                content += "<strong>Speaker </strong> " + (parseInt(index,10)+1) + "<strong> :</strong> " + speaker.name + "<br><br>";
+            }
+            if(speaker.short_biography) {
+                content += "<strong>About the Speaker: </strong><br>" + speaker.short_biography + "<br><br>";
+            }
+            else {
```
Comment (niranjan94) [inline app/static/js/events/scheduler.js]: Please follow proper code-style ... Keep the else in the same line as `}` .. Like `} else {`

File: app/static/js/events/scheduler.js [hunk 4]
```diff
@@ -550,10 +550,32 @@ function addInfoBox($sessionElement, session) {
         html: true,
         title: session.title
     });
-    var speakers = _.map(session.speakers, 'name');
     var content = "";
-    if(speakers.length > 0) {
-        content += "By " + _.join(speakers, ', ') + "<br><br>";
+    if(!_.isNull(session.short_abstract)) {
```
Comment (niranjan94) [inline app/static/js/events/scheduler.js]: Implement this just like speakers ... If there is only long_abstract and no short_abstract, show a truncated version of it.

Comment (Achint08) [inline app/static/js/events/scheduler.js]: Done. :)

File: app/static/js/events/scheduler.js [hunk 5]
```diff
@@ -551,10 +551,44 @@ function addInfoBox($sessionElement, session) {
...
+            content += "<strong>Speaker </strong> " + (parseInt(index, 10)+1) + "<strong> :</strong> " + speaker.name + "<br><br>";
+        }
+        if(speaker.short_biography) {
+            content += "<strong>About the Speaker: </strong><br>" + speaker.short_biography + "<br><br>";
+        } else {
+            session.speakers.long_biography = speaker.long_biography.substr(0, 100);
+            content += "<strong>About the Speaker: </strong><br>" + speaker.long_biography + "<br><br>";
+        }
+    });
+    _.forEach(session.speakers, function(speaker, index) {
```
Comment (niranjan94) [inline app/static/js/events/scheduler.js]: Why is this duplicated ?

Comment (Achint08) [inline app/static/js/events/scheduler.js]: Removed.

**Comment by mariobehling** [comment]
What is missing in this PR?

**Pull request merged**